ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js

```diff
@@ -246,12 +250,15 @@ myAction : function(component, event, helper) {
                 }); */ 
                 this.data.datasets.forEach(function (dataset, i) {
                     var meta = chartInstance.controller.getDatasetMeta(i);
+                    // console.log(meta.data[0]);
                     meta.data.forEach(function (bar, index) {
-                        var model = dataset._meta[Object.keys(dataset._meta)[0]].data[index]._model;                            
-                        var label = model.label;
-                        
-                        ctx.fillText(label, bar._model.x+2, bar._model.y);
-                        
+                        if (bar._datasetIndex === 0) {
```

Comment:
What's this doing?

Reply:
will only fillText label to the right of the first invisible bar, added comment for clarification